src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1313,6 +1313,9 @@ MessageHelper::CreateGlobalPropertiesRequestsToHMI(
       if (app->menu_icon()) {
         ui_msg_params[strings::menu_icon] = (*app->menu_icon());
       }
+      if (app->menu_layout()) {
```

Comment:
the if block that contains this needs the condition `|| app->menu_layout()`

I tested with this patch and all worked.
```
diff --git a/src/components/application_manager/src/message_helper/message_helper.cc b/src/components/application_manager/src/message_helper/message_helper.cc
index 61db5b6ec5..f5f227de4f 100644
--- a/src/components/application_manager/src/message_helper/message_helper.cc
+++ b/src/components/application_manager/src/message_helper/message_helper.cc
@@ -1289,7 +1289,7 @@ MessageHelper::CreateGlobalPropertiesRequestsToHMI(
 
   if (can_send_ui &&
       (app->vr_help_title() || app->vr_help() || app->keyboard_props() ||
-       app->menu_title() || app->menu_icon())) {
+       app->menu_title() || app->menu_icon() || app->menu_layout())) {
     smart_objects::SmartObjectSPtr ui_global_properties = CreateMessageForHMI(
         hmi_apis::messageType::request, app_mngr.GetNextHMICorrelationID());
     if (ui_global_properties) {

```

Reply:
@iCollin nice catch! Fixed in https://github.com/smartdevicelink/sdl_core/pull/3733/commits/0d92a875156c404b9fdab82686de2e4efe05c065